plugins/fluent-bit/integration/tests/tests.bats

```diff
@@ -20,13 +20,14 @@ setup() {
 }
 
 @test "read stdin via fluent-bit" {
+  skip "This test often fails to produce any output when run on GitHub Actions Runners"
```

Comment:
Can we just ditch the test entirely?

Reply:
From past experience I would say: ditch it. We know we can't make this work reliably with the Fluent Bit library, and it's not worth sinking time into this test.

Reply:
I would not remove it entirely, but extend the explanation for the skip to tell the tempted developer to not try to fix this test. That way the knowledge that `fluent-bit stdin` is not reliable enough for this kind of testing is preserved, and nobody wastes time reimplementing a similar test in case we get a bug report for that feature.